### ISSUE TITLE
Update accessibility on Search component

### ISSUE THREAD

**Issue by awentzel** [Microsoft/fast-dna#245]
Search Results component accessibility grade: D. 
https://fluentweb.com/components/search

Need to bring it up to C / green accessibility score.

Feedback from Mat.

Only big improvement to be made here is swapping the `aria-label` for a real `label` element, which we can (visually) hide by pulling in `<Label />`. Worth noting that the `aria-label` approach in use now is _technically_ valid, but iffy in NVDA:
https://www.powermapper.com/tests/screen-readers/labelling/input-text-aria-label/

**Comment by nicholasrice** [comment]
Blocked because this component does not exist yet.

**Comment by awentzel** [comment]
Note this component is also needed by @chrisbalt and team

**Comment by nalogiudice** [comment]
Need guidance on when this component should be created

**Comment by chrisdholt** [comment]
+1 to @nalogiudice as the component does not currently exist. @awentzel I propose that we close this in favor of a new issue to create the search component.

**Comment by chrisdholt** [comment]
closing this as obsolete since there's been no follow-up regarding my suggestion to do so. Accessibility is part of the creation of new components; since "Search" will be a new component, we can address this per usual as part of its creation.
